com.unity.render-pipelines.universal/Documentation~/complex-lit-shader.md

```diff
@@ -1,6 +1,6 @@
 # Complex Lit Shader
 
-The Complex Lit Shader is similar to the Lit shader, but it adds advanced material features. These features often are considerably more resource intensive and therefore are only supported on [Unity Shader Model 4.5](https://docs.unity3d.com/Manual/SL-ShaderCompileTargets.html) hardware. Complex Lit shader is always rendered using forward rendering and used as a fallback shader in deferred rendering. Complex Lit contains all the features that are incompatible with deferred rendering. This Shader uses the most computationally heavy [shading model](shading-model.md) in the Universal Render Pipeline (URP). 
+The Complex Lit Shader inherits the functionality of the Lit shader and adds advanced material features. Some features in this shader might be considerably more resource-intensive and require [Unity Shader Model 4.5](https://docs.unity3d.com/Manual/SL-ShaderCompileTargets.html) hardware. This shader contains the features that are incompatible with the Deferred Rendering Path. URP renders the Complex Lit shader using the Forward Rendering Path, and uses the Complex Lit shader as a fallback shader in the Deferred Rendering Path. 
```

Comment:
Technically there is no inheritance relationship between Lit and ComplexLit. It's technically by convention. We've agreed that ComplexLit features should be a superset of Lit features i.e. it extends Lit features and fallsback to Lit if it is not supported.
I'm not sure if this should affect the documentation wording however.